modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/upgrade-catch-all-check/LPD_36261.testjava

```diff
@@ -12,9 +12,11 @@ import org.osgi.service.component.annotations.Reference;
  */
 public class LPD_36261 {
 
-	public void method() {
-		ExpandoColumnLocalServiceUtil.getColumn(123, "test123");
-		_expandoColumnLocalService.getColumn(123, "test123");
+	public void method(long companyId, String className) {
+
+		ExpandoColumnLocalServiceUtil.getColumn(companyId, className);
+		_expandoColumnLocalService.getColumn(companyId, className);
+		_expandoColumnLocalService.getColumn(null, "test123");
```

Comment:
message cases, only one attribute must be the message trigger @regiCesar21. And the default value for long isn't null, is 123.